test/test_data/jre.py

```diff
@@ -0,0 +1,11 @@
+mapping_test_data = [
+    {"product": "jre", "version": "1.0.0", "version_strings": ["openjdk 1.0.0"]}
```

Comment:
It's erroring out because your product is being detected as 'java' rather than 'jre'

```
>           assert product in list_products
E           AssertionError: assert 'jre' in {'java'}
```

You could easy fix the tests by changing 'jre" to 'java" but I'm not convinced that's the right choice for a jre checker, so you might want to fix these by changing the product above.

Reply:
changed it to `JRE` in the products of `JRE`